addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -59,6 +59,7 @@ var MassMailingFieldHtml = FieldHtml.extend({
         var $editable = this.wysiwyg.getEditable();
 
         return this.wysiwyg.saveCroppedImages(this.$content).then(function () {
+            self.wysiwyg.removeCacheBusters(self.$content);
```

Comment:
Small error in commit message: 
- "This commit changes both of these things:" -> then you mention 3 things
- Checkbox to show optimized: mention it is in debug mode :+1: